src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(callback) {
+    const compareFunction = callback
+    || ((a, b) => String(a).localeCompare(String(b)));
```

Comment:
The reason the test is failing is this default compare function. 
It is better to use something like this ` (a, b) => a.toString() > b.toString()`